k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -2994,7 +2995,9 @@ public void run() {
                     } finally {
                         clearSendingNotificationIfNecessary(account);
                     }
+
                 }
+                Toast.makeText(context,"message envoyée ", Toast.LENGTH_SHORT).show();
```

Comment:
This needs to: 

* Be optional, based on a preferences setting
* Use a field from R.java so that it is translated correctly.